app/controllers/suggestions_controller.rb

```diff
@@ -0,0 +1,116 @@
+require_relative '../../lib/TunesTakeoutWrapper.rb'
+require 'yelp'
+require 'httparty'
+
+class SuggestionsController < ApplicationController
+#skip before action
+
+  def index
+    #shows top 20 suggestions, ranked by total number of favorites
+    suggestion_ids = TunesTakeoutWrapper.top["suggestions"] #this should return array of sugg ids
+
+    @pairings = []
+    suggestion_ids.each do |suggestion_id|
+
+      suggestion_object = TunesTakeoutWrapper.find(suggestion_id)
+
+      suggestion = suggestion_object["suggestion"]
+
+      yelp_id = suggestion["food_id"]
+      spotify_id = suggestion["music_id"]
+      spotify_type = suggestion["music_type"]
+
+      if current_user
+        favorite = is_favorite?(suggestion["id"])
+      end
+
+
+      array = []
+      array << Food.find(yelp_id)  #first thing in array is yelp
+      array << Music.find(spotify_id, spotify_type) #second thing is music
+      array << suggestion["id"]
+      if favorite
+        array << favorite
+      end 
+
```

Comment:
I'm not the biggest fan of calling arrays "array", just because it's a special word